app/javascript/pages/components/Calendar.jsx

```diff
@@ -194,9 +204,28 @@ class Calendar extends React.Component {
   }
 
   render() {
+    const back = "< Back"
     return (
       <section className="route Calendar">
-        <div className="jobs" />
+        <div className="container">
+          <a href="/" className="back-link">{back}</a>
+          <div className="jobs__title">Employment by Industry</div>
```

Comment:
A few refactorings to consider:
- Changing the `div` on line 212 to an `h1` heading
- Changing the `div` on line 215 to an `h2` heading (still overriding the font size for the year in the styling, though)
- Merging the divs on lines 221 and 222 into one div with both `container` and `jobs__explanation` classes

Reply:
Done